Misc/NEWS.d/next/Library/2023-04-07-15-09-26.gh-issue-74690.0f886b.rst

```diff
@@ -0,0 +1,3 @@
+The members of a runtime-checkable protocol are now considered "frozen" at
+runtime as soon as the class has been created. See
+:ref:`"What's new in Python 3.12" <whatsnew-typing-py312>`for more details.
```

Comment:
```suggestion
:ref:`"What's new in Python 3.12" <whatsnew-typing-py312>` for more details.
```

This should fix the syntax error.

Reply:
You found it faster but I committed it faster :)